pkg/core/inference_service_test.go

```diff
@@ -17,7 +17,7 @@ func (suite *CoreTestSuite) TestCreateInferenceService() {
 	// create mode registry service
 	service := suite.setupModelRegistryService()
 
-	parentResourceId := suite.registerServingEnvironment(service, nil, nil)
+	parentResourceId := suite.registerServingEnvironment(service, "", nil)
```

Comment:
I think this could have been a placeholder string, instead of the empty literal, but I would possibly review this as part of a follow-up PR for the tests